frontend/src/views/producer/dashboard/edition-form/TicketCatalogInputGroup.jsx

```diff
@@ -78,7 +78,6 @@ const TicketCatalogInputGroup = props => {
               'ticketDescription',
             )}
             value={ticketType.ticketDescription}
-            disabled={props.disabled}
```

Comment:
Why do we need to remove this attribute? We could just control this using props. Just change the `disabled` props value.